datafusion/common/src/scalar.rs

```diff
@@ -3437,10 +3437,6 @@ mod tests {
             ScalarValue::Decimal128(None, 10, 2),
             ScalarValue::try_from_array(&array, 3).unwrap()
         );
-        assert_eq!(
```

Comment:
See https://github.com/apache/arrow-datafusion/pull/5441#discussion_r1121792711

This test was wrong